src/routes/main.js

```diff
@@ -47,6 +47,9 @@ export default [
         { translation: 'JOINGROUP.ALL_GROUPS', route: { name: 'groupsGallery' } },
         { type: 'activeGroupInfo' },
       ],
+      beforeEnter: 'groups/selectGroupInfo',
+      onError: 'group id not found',
```

Comment:
There could be other reasons for an error... do we not get a reasonable message from the error itself?